openapi/paths/permissions-emulation.yaml

```
@@ -0,0 +1,75 @@
+post:
+  tags:
+    - Profile
+  summary: Start permissions emulation
+  operationId: PostPermissionsEmulation
+  description: |
+    Start permissions emulation.
+
+    Emulation during emulation is not supported. If request sent during an ongoing emulation 403 sent in response.
```

```suggestion
    Emulation during emulation is not supported. If request sent during an ongoing emulation 403 sent in response.
    
    Escalation is also not supported.
```

Am I correct that escalation is not supported?

---

Yes, sure.